Tests/UnitTests/PlayMode/CardSets/UldumCardsGenTests.cpp

```diff
@@ -3958,6 +3958,47 @@ TEST_CASE("[Neutral : Minion] - ULD_250 : Infested Goblin")
     CHECK_EQ(curHand[5]->card->name, "Scarab");
 }
 
+// --------------------------------------- MINION - WARRIOR
+// [ULD_253] Tomb Warden - COST:8 [ATK:3/HP:6]
+// - Race: Mechanical, Set: Uldum, Rarity: Rare
+// --------------------------------------------------------
+// Text: <b>Taunt</b>
+//       <b>Battlecry:</b> Summon a copy of this minion.
+// --------------------------------------------------------
+// GameTag:
+// - TAUNT = 1
+// - BATTLECRY = 1
+// --------------------------------------------------------
+TEST_CASE("[Warrior : Minion] - ULD_253 : Tomb Warden")
+{
+    GameConfig config;
+    config.player1Class = CardClass::WARRIOR;
+    config.player2Class = CardClass::MAGE;
+    config.startPlayer = PlayerType::PLAYER1;
+    config.doFillDecks = false;
+    config.autoRun = false;
+
+    Game game(config);
+    game.Start();
+    game.ProcessUntil(Step::MAIN_ACTION);
+
+    Player* curPlayer = game.GetCurrentPlayer();
+    Player* opPlayer = game.GetOpponentPlayer();
+    curPlayer->SetTotalMana(10);
+    curPlayer->SetUsedMana(0);
+    opPlayer->SetTotalMana(10);
+    opPlayer->SetUsedMana(0);
+
+    auto& curField = *(curPlayer->GetFieldZone());
+    const auto card =
+        Generic::DrawCard(curPlayer, Cards::FindCardByName("Tomb Warden"));
+
+    game.Process(curPlayer, PlayCardTask::Minion(card));
+
+    CHECK_EQ("Tomb Warden", curField[0]->card->name);
+    CHECK_EQ("Tomb Warden", curField[1]->card->name);

```

Comment:
```suggestion
    auto& curField = *(curPlayer->GetFieldZone());
    
    const auto card1 =
        Generic::DrawCard(curPlayer, Cards::FindCardByName("Tomb Warden"));

    game.Process(curPlayer, PlayCardTask::Minion(card1));
    CHECK_EQ(curField.GetCount(), 2);
    CHECK_EQ(curField[0]->card->name, "Tomb Warden");
    CHECK_EQ(curField[1]->card->name, "Tomb Warden");
```

And could you please add code to check if its attack/health copied well when summoning a buffed minion?

Reply:
To achieve it, it needs cards able to give a buffer to a minion in the hand zone. Are there cards having the effect? I will try to find one among the all cards but it will be more fast than it if you let me know the one. 🙏

Reply:
I recommend 'Galakrond, the Unbreakable'.
https://hearthstone.gamepedia.com/Galakrond,_the_Unbreakable

Reply:
You don't have to use only cards from the same card class.

Reply:
I applied it at e4aca81. Could you review again?